web/text_layer_builder.js

```diff
@@ -350,7 +354,9 @@ var TextLayerBuilder = (function TextLayerBuilderClosure() {
       });
       div.addEventListener('mouseup', function (e) {
         if (self.enhanceTextSelection && self.textLayerRenderTask) {
-          self.textLayerRenderTask.expandTextDivs(false);
+          expandDivsTimer = window.setTimeout(function() {
+            self.textLayerRenderTask.expandTextDivs(false);
+          }, 300);
```

Comment:
Make the value 300 a constant named `EXPAND_TEXT_DIVS_TIMEOUT` and add it here: https://github.com/jeremypress/pdf.js/blob/eebbc2d4034781ef9bab93a0ae6ce90481d11853/web/text_layer_builder.js#L30